src/graphics/Screen.cpp

```diff
@@ -305,12 +305,25 @@ static bool haveGlyphs(const char *str)
 /// Used on eink displays while in deep sleep
 static void drawDeepSleepScreen(OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x, int16_t y)
 {
+    const char *idText = owner.short_name;
+
     // Next frame should use full-refresh, and block while running, else device will sleep before async callback
     EINK_ADD_FRAMEFLAG(display, COSMETIC);
     EINK_ADD_FRAMEFLAG(display, BLOCKING);
 
     LOG_DEBUG("Drawing deep sleep screen\n");
-    drawIconScreen("Sleeping...", display, state, x, y);
+    
+    // Check if idText is not assigned OR empty OR has non printable characters
+    if (!idText || strlen(idText) == 0 || !haveGlyphs(idText)) {
```

Comment:
I think `!idText` and `strlen(idText)==0` are technically checking the same thing (gotta love C++). I think maybe it's not so important to check this though: `owner.short_name` is assigned when the node is initially set up, and if something goes wrong, we probably won't catch with `strlen(idText) == 0` anyway.

Reply:
> For logical naming - I would like to change the name of the functinon into haveNoGlyphs - since a true answer would mean exactly this.

I wouldn't change it to `haveNoGlyhps`. The reason it was called `haveGlyphs` is because it checks if our font has all the "glyphs" (characters) needed to print the string. It returns false if we're missing a glyph which we need.

The fact that I had to explain this though suggests that the name probably isn't clear enough! Maybe `fontHasChars(idText)` or something similar?

Reply:
OK. I think I got the use from reading the code - just thought the name would be more consistent as your fontHasChars. I leave it for now.